### PR TITLE
v1.x rollup

### PR DESCRIPTION
**Pull request by rhinof**

### CODE REVIEW

File: gbus/tx/mysql/txprovider.go [hunk 1]
```diff
@@ -5,12 +5,11 @@ import (
 	"time"
 
 	_ "github.com/go-sql-driver/mysql" //blank import
-	"github.com/wework/grabbit/gbus"
 	"github.com/wework/grabbit/gbus/tx"
 )
 
 //NewTxProvider returns a new PgProvider
-func NewTxProvider(connStr string) (gbus.TxProvider, error) {
+func NewTxProvider(connStr string) (*tx.Provider, error) {
```
Comment (vladshub) [inline gbus/tx/mysql/txprovider.go]: @rhinof do you want to remove the use of the interface here?

**Comment by vladshub** [comment]
There are some conflicts...

**Comment by rhinof** [comment]
yep, working on the conflicts now

**Comment by rhinof** [comment]
Resolved conflicts via another PR. this PR is canceled

**Pull request closed without merging**